src/main/java/com/example/bytex/user/UserEntityController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.bytex.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "users")
+public class UserEntityController {
+
+    UserEntityService userEntityService;
```

Comment:
Is there a reason for not making this private? 🤔

Reply:
yep, it's called "prajeala".